src/tests/simulation/tests/integration_tests/recovery/background_ddl.rs

```diff
@@ -86,14 +112,57 @@ async fn test_background_mv_barrier_recovery() -> Result<()> {
 
     sleep(Duration::from_secs(10)).await;
 
-    // Make sure after finished, we should have 5000_000 rows.
     session
         .run("SELECT COUNT(v1) FROM m1")
         .await?
-        .assert_result_eq("600000");
+        .assert_result_eq("700000");
+
+    // Make sure that if MV killed and restarted
+    // it will not be dropped.
 
     session.run("DROP MATERIALIZED VIEW m1").await?;
     session.run("DROP TABLE t1").await?;
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_background_ddl_cancel() -> Result<()> {
```

Comment:
Hi @wangrunji0408 , could you PTAL at this test? I would expect that it either completes really quickly, or will hit timeout errors.

But it actually takes a long time to run:
```
        SLOW [>  5.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 10.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 15.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 20.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 25.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 30.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 35.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 40.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 45.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
        SLOW [> 50.000s] risingwave_simulation::integration_tests recovery::background_ddl::test_background_ddl_cancel
     Running [ 00:00:52] [                                                                                                                                                                                                                                             ]  0/1 : 1 running, 0 passed, 6 skipped        Canceling due to interrupt: 1 tests still running
```

This seems strange to me. Does `timeout` not work in madsim?